src/Faker/Provider/Imei.php

```diff
@@ -0,0 +1,23 @@
+<?php
+
+namespace Faker\Provider;
+
+use Faker\Calculator\Luhn;
+
+class Imei extends Base
```

Comment:
Let's consider that `PhoneNumber` is named `Phone` (which should have been the case from the beginning), so yes, this is necessary.